src/cdk/v2/destinations/rakuten/utils.js

```diff
@@ -52,14 +64,8 @@ const constructLineItems = (properties) => {
   });
 
   // Map 'amountList' by evaluating 'amount' or deriving it from 'price' and 'quantity'
-  const amountList = products.map((product) => {
-    if (!product?.amount && !product?.price) {
-      throw new InstrumentationError('Either amount or price is required for every product');
-    }
-    return product.amount * 100 || (product.quantity || 1) * 100 * product.price;
```

Comment:
If both amount & price are in the payload, Is this calculation considering both amount & price is valid ? cc: @anantjain45823

Reply:
Yes we are giving priority to amount over qty * price

Reply:
sorry, I've overseen the or condition